### PR TITLE
Update troubleshooting docs. Deploy docs on master commits.

### PR DESCRIPTION
**Pull request by laurenfrederick**

### CODE REVIEW

File: docs/troubleshooting/troubleshoot_deployment.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ hide_title: true
 
 This document provides 'notes' on frequently encountered deployment issues. The issues reported are organized by relevant subsection.
 
+## Terraform Init
```
Comment (markdboyd) [inline docs/troubleshooting/troubleshoot_deployment.md]: This isn't really specific to `terraform init`. I would make the header `Add logging output` or something

File: docs/troubleshooting/troubleshoot_deployment.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ hide_title: true
...
+
+The `TF_LOG` environment variable can be set to help debug Terraform-specific issues. See [Terraform debugging](https://www.terraform.io/docs/internals/debugging.html).
```
Comment (markdboyd) [inline docs/troubleshooting/troubleshoot_deployment.md]: I would specify that setting `TF_LOG` will log verbose output from Terraform commands to help debug issues

**Comment by laurenfrederick** [comment]
@markdboyd Ready for re-review

File: docs/troubleshooting/troubleshoot_deployment.md [hunk 3]
```diff
@@ -8,6 +8,20 @@ hide_title: true
 
 This document provides 'notes' on frequently encountered deployment issues. The issues reported are organized by relevant subsection.
 
+## Terraform Logging
+
+The `TF_LOG` environment variable can be set to help debug Terraform-specific issues. See [Terraform debugging](https://www.terraform.io/docs/internals/debugging.html).
+
+For example `export TF_LOG=DEBUG` will log verbose output from Terraform commands to help debug issues.
+
+## Terraform Init
```
Comment (markdboyd) [inline docs/troubleshooting/troubleshoot_deployment.md]:
```suggestion
## `terraform knit`
```

Total nitpick. Feel free to take it or leave it

**Pull request merged**